website/src/pages/index.tsx

```diff
@@ -191,12 +191,12 @@ function Feature({
 
   return (
     <div className={clsx('col', className)}>
-      <img
+      <Image
         className={styles.featureImage}
         alt={feature.title}
         width={Math.floor(feature.image.width)}
         height={Math.floor(feature.image.height)}
-        src={withBaseUrl(feature.image.src)}
+        img={withBaseUrl(feature.image.src)}
```

Comment:
the ideal image plugin is not designed to work with SVGs

Eventually we could import the svgs and use them as components instead of using `<img src="xyz.svg"/>` like we do atm

Reply:
> the ideal image plugin is not designed to work with SVGs

I noticed that it only works with JPEGs and PNGs but somehow completely missed out in this case 😅

> Eventually we could import the svgs and use them as components instead of using `<img src="xyz.svg"/>` like we do atm

@slorber Should I address this change in this PR?